### ISSUE TITLE
Please publish the full Evergreen distribution updates of WebView2 to Chocolatey.

### ISSUE THREAD

**Issue by noseratio** [MicrosoftEdge/WebView2Feedback#1591]
Please consider publishing the full Evergreen distribution updates of WebView2 runtime to [Chocolatey](https://chocolatey.org/). 

This would make it easy to set it up for CI pipelines, build agents and other DevOps scenarios.

**Comment by champnic** [comment]
Thanks for the request Andrew! I've added this scenario to our backlog.

**Comment by champnic** [comment]
@noseratio Have you looked into the nuget command line for your pipeline? https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference

**Comment by noseratio** [comment]
> @noseratio Have you looked into the nuget command line for your pipeline? https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference

@champnic, IIUIC Nuget is good for getting the [`Microsoft.Web.WebView2` .NET package](https://www.nuget.org/packages/Microsoft.Web.WebView2), but for installing the [language-independent WebView2 runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section) binaries in shared mode (that requires admin rights). 

Am I missing something? Is there another NuGet package for the runtime itself? I mean, that might be an option (after all, Chocolatey uses Nuget format for its packages), but I think that'd be quite uncommon.

On a side note, I  use `dotnet restore` to get the .NET bits, not even sure if it uses Nuget CLI under the hood.

**Comment by champnic** [comment]
@noseratio Oh sorry I completely misunderstood the request! You are right, we only have the SDK in Nuget and don't ship the runtime through other means.

The CI should still be able to deploy the runtime using the bootstrapper download link. Or are you trying to install specific versions?

**Comment by noseratio** [comment]
> The CI should still be able to deploy the runtime using the bootstrapper download link. Or are you trying to install specific versions?

@champnic, yep, it'd be great to be able to install a specific version of full shared runtime into a clean container (for predictable builds & test results). My personal preference would be to do it via Chocolatey, but WinGet would work too.  

Someone from the community is already doing that for Edge itself: https://community.chocolatey.org/packages/microsoft-edge

I realize I could use a specific "fixed version" of WV2 runtime, but that's not the distribution model we use for production and thus it'd be less than ideal to use it for CI.

**Comment by tongyuze** [comment]
@champnic 
I have the same question here. I want to test our library with webview2 integrated in the azure pipeline. But for the pipeline's virtual machine, we need to download and install the Evergreen WebView2 runtime. 
Is there a method to do this by using a yaml file?

**Comment by champnic** [comment]
@tongyuze I'm not super familiar with Azure Pipeline Yaml, but it should be able to run powershell scripts that you could use to download the runtime installer and run it.
